doc/vlan/SONiC_vlan_translation_qnq.md

```diff
@@ -0,0 +1,1025 @@
+  add         Configure VLAN stacking match customer VLANs on an interface
+  remove      Remove VLAN stacking match customer VLANs on an interface
+  VLAN-LIST   (-) or (,) separated individual VLAN IDs and ranges of VLAN IDs; for example, 20,70-100,142
+
+sonic(conf-if-Ethernet4)# switchport vlan-mapping remove 
+  VLAN-LIST   (-) or (,) separated individual VLAN IDs and ranges of VLAN IDs; for example, 20,70-100,142
+
+sonic(conf-if-Ethernet4)# switchport vlan-mapping remove 50 
+  dot1q-tunnel  Configure 801.1Q tunneling (Q-in-Q)
+
+sonic(conf-if-Ethernet4)# switchport vlan-mapping remove 50 dot1q-tunnel 
+  <1..4094>  Configure service provider VLAN ID
+
+sonic(conf-if-Ethernet4)# switchport vlan-mapping remove 50 dot1q-tunnel 100 
+  <cr>  
+
+sonic(conf-if-Ethernet4)# switchport vlan-mapping 
+  <1..4094>   Configure customer VLAN ID
+  add         Configure VLAN stacking match customer VLANs on an interface
+  remove      Remove VLAN stacking match customer VLANs on an interface
+  VLAN-LIST   (-) or (,) separated individual VLAN IDs and ranges of VLAN IDs; for example, 20,70-100,142
+
+sonic(conf-if-Ethernet4)# no switchport 
+  access        Remove access mode characteristics of the interface
+  allowed       Remove all VLANs configured on the interface
+  trunk         Remove trunking parameters on an interface
+  vlan-mapping  Remove VLAN translation parameters on an interface
+
+sonic(conf-if-Ethernet4)# no switchport vlan-mapping 
+  <1..4094>     Configure service provider VLAN ID
+  dot1q-tunnel  Configure 801.1Q tunneling (Q-in-Q)
+
+sonic(conf-if-Ethernet4)# no switchport vlan-mapping dot1q-tunnel 
+  <1..4094>  Configure service provider VLAN ID
+
+sonic(conf-if-Ethernet4)# no switchport vlan-mapping dot1q-tunnel 200 
+  <cr>  
+```
+
+#### 3.6.2.1.3 Configuration restrictions
+
+
+* An SVLAN cannot be part of more than one VLAN translation (single or double tagged) on a given Interface
+* AN SVLAN cannot be part of both VLAN stacking and translation scheme on a given Interface
+* Only Layer2 traffic is supported on an SVLAN. So no Layer3 configuration is supported on SVLAN
+
+
+
+#### 3.6.2.2 Show Commands
+
+#### 3.6.2.2.1 VLAN Translation
+
+```
+Command:
+			show interface vlan-mappings 
+```
+Shows all the VLAN translation mappings configured in the system
+
+```
+show interface vlan-mappings 
+-------------------------------------------------------------------
+Name           Outer           Inner                  Mapped Vlan  
+-------------------------------------------------------------------
+Eth1/1         100             -                       1000        
+Eth1/1         200             20                      2000        
+Eth1/7         100             -                       1000        
+```
+
+```
+Command:
+		 	show interface {Ethernet [port] | PortChannel [po-id] } vlan-mappings
+```		 
+
+Shows all the VLAN translation mappings configured on a given interface
+
+```
+show interface Eth1/1 vlan-mappipngs
```

Comment:
Please provide output with Port-Channel interface also

Reply:
Will capture this.